include/mal_log/util/system.hpp

```diff
@@ -115,9 +115,11 @@ either expressed or implied, of Rafael Gago Castano.
         const char fs_separator = '/';
     }
 #else
-    namespace mal {
-        const char fs_separator = '\\';
-    }
+	#ifndef _MSC_VER
```

Comment:
I can't grasp what is this for, but I guess that no case should leave "fs_separator" undefined.

Reply:
Well, at line 86 you already define the fs_seperator if we are using Visual Studio.
You define it again here; hence, I get a redefinition error. with this workaround, if you compile with sth else other than VS, you define fs_seperator here.